tests/rptest/tests/nodes_decommissioning_test.py

```diff
@@ -490,7 +492,6 @@ def calculate_total_learners_gap() -> int | None:
 
         self.logger.info(f"decommissioning node: {to_decommission_id}", )
         self._decommission(to_decommission_id)
-        self.producer.wait()
```

Comment:
why is wait removed from there? this changes what the test does no? previously the test waited for consumer to finish before proceeding. but now?